docs/csp-metrics-api.md

```
@@ -0,0 +1,494 @@
+async function monitorViolations() {
+  const metrics = await fetch('/api/csp-metrics?timeRange=1h');
+  const data = await metrics.json();
+  
+  if (data.data.summary.highRiskViolations > 0) {
+    console.warn('High-risk CSP violations detected:', 
+                 data.data.summary.highRiskViolations);
+    
+    // Get violation details
+    const details = await fetch('/api/csp-metrics?includeDetails=true');
+    const violations = await details.json();
+    
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)